glean_parser/metrics.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         gecko_datapoint: str = "",
         no_lint: Optional[List[str]] = None,
         data_sensitivity: Optional[List[str]] = None,
-        _config: Optional[Dict[str, Any]] = None,
+        _config: Dict[str, Any] = {},
```

Comment:
Python nonsense: Shouldn't use a mutable data structure as a default keyword argument because it will use the same object for every invocation.

The usual workaround pattern is to set `None` as the default and then have:

```python
if _config is None:
    _config = {}
```

in the body.

Reply:
(Aside -- the `bugbear` linter slated to turn on in #222 would have caught this sort of thing...)